### ISSUE TITLE
Player test are passing but they are not testing

### ISSUE THREAD

**Issue by eliawk** [Paratii-Video/paratii-portal#105]
this test pass
```
it('play a video @watch', async function () {
    // check sanity - the video should be available now
    assert.isOk(await paratii.core.vids.get(videoId))

    browser.url(`http://localhost:8080/play/${videoId}`)
    browser.waitAndClick('#player')
    browser.waitAndClick('.media-control')
  })
```
even if there is no `.media-control` element in the browser.

try to console log
```
const player = browser.waitForExist('#player', 5000)
console.log('player', player)
 ```

and i get always `player { state: 'pending' }`

**Comment by jellegerbrandy** [comment]
looks like "player" is  a promise here (which is pending), you'd need to sprinke some `await `s in the code perhaps.

Did not try it, but can you try:
    const player = await browser.waitForExait('#player`)

**Comment by bent0b0x** [comment]
Working on fixing this on my branch

**Comment by bent0b0x** [comment]
This is resolved by #110
